src/accessibility-checker-app/index.js

```diff
@@ -371,24 +371,24 @@ class AccessibilityCheckerHighlight {
 	addHighlightPanel() {
 		const newElement = `
 			<div class="edac-highlight-panel">
-			<button id="edac-highlight-panel-toggle" class="edac-highlight-panel-toggle" title="Toggle accessibility tools"></button>
-			<div id="edac-highlight-panel-description" class="edac-highlight-panel-description" tabindex="0">
+			<button id="edac-highlight-panel-toggle" class="edac-highlight-panel-toggle" aria-haspopup="dialog">Accessibility Checker Tools</button>
```

Comment:
A better name to describe the button. Let screen readers know the button opens a dialog.